gateway/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM --platform=${BUILDPLATFORM:-linux/amd64} teamserverless/license-check:0.3.9 as license-check
 
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.13 as build
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.15 as build
 
-ENV GO111MODULE=off
+ENV GO111MODULE=on
```

Comment:
If you change the working directory to `/app` then this isn't needed anymore

Reply:
I will bear that in mind for the future. Thank you